README.md

```diff
@@ -45,9 +45,8 @@ Alternatively, use Brooklyn as an integrated-stand-alone management node for you
 Three quick start options are available:
 
 * The [getting started guide](https://brooklyn.incubator.apache.org/v/latest/start/running.html) will step you through downloading and installing Brooklyn and running the examples.
-* Alternatively, [download the latest release](https://github.com/brooklyncentral/brooklyn/tarball/master) (tgz),
-* or, fork or clone the repo: `git clone git://github.com/apache/incubator-brooklyn.git` then `cd incubator-brooklyn; mvn clean install; cd usage/dist/target/brooklyn-dist; bin/brooklyn launch`
-`.
+* Alternatively, [download the latest release](https://github.com/brooklyncentral/brooklyn/tarball/master) (tgz).
```

Comment:
Ouch!  This needs to point at `apache/incubator-brooklyn/tarball/master` not `brooklyncentral/brooklyn` !!!  I'll fix when I merge. Could explain why you got that weird `external-location....txt` error.